### ISSUE TITLE
Support Wikimedia Commons

### ISSUE THREAD

**Issue by dpriskorn** [TeamNewPipe/NewPipeExtractor#609]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

I would like it to support Wikimedia Commons and the different file versions available as seen in the photo.
![image](https://user-images.githubusercontent.com/68460690/114377169-a4ef4800-9b86-11eb-810b-2925cabcf142.png)

https://commons.wikimedia.org/wiki/File:Data_imports_for_Wikidata_Lexemes_(panel_discussion_30_Lexicodays).webm

#### How will you/everyone benefit from this feature?

This feature will help users discover valuable content on Commons and perhaps contribute content themselves.

**Comment by prototyperspective** [comment]
Yes this would be awesome. There are over 350,000 free-licensed videos on Commons, all well organized via categories and the official Commons app has nearly no users and can't play videos. For example, I'd like to play audio files on Commons with NewPipe so that I can listen [to Spoken Wikipedia audios](https://commons.wikimedia.org/wiki/Category:Spoken_Wikipedia_articles_using_English-language_speech_synthesis) without having to first find & download them separately and then moving them into my podcast player app.

I made a more detailed proposal for this here, this really would be a big win-win: **https://meta.wikimedia.org/wiki/Community_Wishlist/Wishes/Add_support_for_Wikimedia_Commons_in_the_open_source_NewPipe_media_player_app**
